src/menus/MainMenu.java

```diff
@@ -0,0 +1,59 @@
+package menus;
+
+import java.awt.Color;
+import java.awt.Font;
+import java.util.ArrayList;
+
+import DLibX.*;
+import utils.Gamepad;
+import net.java.games.input.Controller;
+
+public class MainMenu extends Menu {
+
+    public MainMenu(DConsole dc, ArrayList<Gamepad> gamepads, Controller[] controllers) {
+        super(dc, gamepads, controllers);
+    }
+
+    /**
+     * Runs the main menu screen
+     * 
+     * @return null
```

Comment:
This return line could probably be removed, but it doesn't affect the code at all so it can be merged in.

Reply:
ya idk why I did that, I do not know javadocs lol.